sympy/solvers/ode/tests/test_ode.py

```diff
@@ -3527,8 +3517,204 @@ def test_issue_14395():
+    #      (8*t*exp(12*t) + 5*exp(12*t) - 5)*exp(4*t)/4,
+    #      (exp(12*t) - 1)*exp(4*t)/2],
+    #     [(-8*t*exp(12*t) - exp(12*t) + 1)*exp(4*t)/4,
+    #      (-8*t*exp(12*t) - exp(12*t) + 5)*exp(4*t)/4,
+    #      (-exp(12*t) + 1)*exp(4*t)/2],
+    #     [4*t*exp(16*t), 4*t*exp(16*t), exp(16*t)]])
+    expAt = Matrix([
+        [2*t*exp(16*t) + 5*exp(16*t)/4 - exp(4*t)/4, 2*t*exp(16*t) + 5*exp(16*t)/4 - 5*exp(4*t)/4,  exp(16*t)/2 - exp(4*t)/2],
+        [ -2*t*exp(16*t) - exp(16*t)/4 + exp(4*t)/4,  -2*t*exp(16*t) - exp(16*t)/4 + 5*exp(4*t)/4, -exp(16*t)/2 + exp(4*t)/2],
+        [                             4*t*exp(16*t),                                4*t*exp(16*t),                 exp(16*t)]
+        ])
+    assert matrix_exp(A, t) == expAt
+
+    A = Matrix([[1, 1, 0, 0],
+                [0, 1, 1, 0],
+                [0, 0, 1, -S(1)/8],
+                [0, 0, S(1)/2, S(1)/2]])
+    expAt = Matrix([
+        [exp(t), t*exp(t), 4*t*exp(3*t/4) + 8*t*exp(t) + 48*exp(3*t/4) - 48*exp(t),
+                            -2*t*exp(3*t/4) - 2*t*exp(t) - 16*exp(3*t/4) + 16*exp(t)],
+        [0, exp(t), -t*exp(3*t/4) - 8*exp(3*t/4) + 8*exp(t), t*exp(3*t/4)/2 + 2*exp(3*t/4) - 2*exp(t)],
+        [0, 0, t*exp(3*t/4)/4 + exp(3*t/4), -t*exp(3*t/4)/8],
+        [0, 0, t*exp(3*t/4)/2, -t*exp(3*t/4)/4 + exp(3*t/4)]
+        ])
+    assert matrix_exp(A, t) == expAt
+
+    A = Matrix([
+    [ 0, 1,  0, 0],
+    [-1, 0,  0, 0],
+    [ 0, 0,  0, 1],
+    [ 0, 0, -1, 0]])
+
+    expAt = Matrix([
+    [ cos(t), sin(t),         0,        0],
+    [-sin(t), cos(t),         0,        0],
+    [      0,      0,    cos(t),   sin(t)],
+    [      0,      0,   -sin(t),   cos(t)]])
+    assert matrix_exp(A, t) == expAt
+
+    A = Matrix([
+    [ 0, 1,  1, 0],
+    [-1, 0,  0, 1],
+    [ 0, 0,  0, 1],
+    [ 0, 0, -1, 0]])
+
+    expAt = Matrix([
+    [ cos(t), sin(t),  t*cos(t), t*sin(t)],
+    [-sin(t), cos(t), -t*sin(t), t*cos(t)],
+    [      0,      0,    cos(t),   sin(t)],
+    [      0,      0,   -sin(t),   cos(t)]])
+    assert matrix_exp(A, t) == expAt
+
+    # This case is unacceptably slow right now but should be solvable...
+    #a, b, c, d, e, f = symbols('a b c d e f')
+    #A = Matrix([
+    #[-a,  b,          c,  d],
+    #[ a, -b,          e,  0],
+    #[ 0,  0, -c - e - f,  0],
+    #[ 0,  0,          f, -d]])
+
+    A = Matrix([[0, I], [I, 0]])
+    expAt = Matrix([
+    [exp(I*t)/2 + exp(-I*t)/2, exp(I*t)/2 - exp(-I*t)/2],
+    [exp(I*t)/2 - exp(-I*t)/2, exp(I*t)/2 + exp(-I*t)/2]])
+    assert matrix_exp(A, t) == expAt
+
+
 def test_sysode_linear_neq_order1():
-    from sympy.abc import t
+
+    x, y = symbols('x y', cls=Function)
+    a, b, c, t = symbols('a b c t')
+
+    # The 2x2 systems tested below are all handled by _linear_2eq_order1_type1
+    # rather than _linear_neq_order1_type1. I'm adding the tests here because
+    # I think that _linear_2eq_order1_type1 should be removed in favour of
+    # _linear_neq_order1_type1.
```

Comment:
Is this comment still true?

Reply:
Are these examples now duplicated in `test_systems`?

Reply:
Some of them are duplicated I think. What should I do, remove those ones from ```test_ode.py```

Reply:
Yes. Eventually all of the examples should be removed from `test_ode.py`.